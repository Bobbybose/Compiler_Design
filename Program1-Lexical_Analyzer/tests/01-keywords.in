if
else
while
do
for
int
float
char
void
return
struct
if else while do for int float char void return struct