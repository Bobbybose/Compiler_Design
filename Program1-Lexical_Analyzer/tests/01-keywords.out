if: 1, 1
else: 2, 1
while: 3, 1
do: 4, 1
for: 5, 1
int: 6, 1
float: 7, 1
char: 8, 1
void: 9, 1
return: 10, 1
struct: 11, 1
if: 12, 1
else: 12, 4
while: 12, 9
do: 12, 15
for: 12, 18
int: 12, 22
float: 12, 26
char: 12, 32
void: 12, 37
return: 12, 42
struct: 12, 49